pkg/pull/pull.go

```diff
@@ -283,27 +275,30 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 	}
 	log.FinishSpinner()
 
-	var newHelmCharts []*kotsv1beta1.HelmChart
-	if !pullOptions.SkipHelmChartCheck {
-		renderDir := pullOptions.RootDir
-		if pullOptions.CreateAppDir {
-			renderDir = filepath.Join(pullOptions.RootDir, u.Name)
-		}
+	renderDir := pullOptions.RootDir
+	if pullOptions.CreateAppDir {
+		renderDir = filepath.Join(pullOptions.RootDir, u.Name)
+	}
 
-		newHelmCharts, err = kotsutil.LoadHelmChartsFromPath(renderDir)
+	newHelmCharts, err := kotsutil.LoadHelmChartsFromPath(renderDir)
```

Comment:
this needed to be set to properly set the UseHelmInstall option argument to the FindPrivateImages function

https://github.com/replicatedhq/kots/blob/e8c349031e2b43483431fa9e1edc5837a78c5442/pkg/pull/pull.go#L429